econml/cate_estimator.py

```diff
@@ -662,6 +668,7 @@ def _get_inference_options(self):
         # add debiasedlasso to parent's options
         options = super()._get_inference_options()
         options.update(debiasedlasso=LinearModelFinalInference)
+        options.update(auto=LinearModelFinalInference)
```

Comment:
Should be unnecessary given the change to the mixin, no?

Reply:
Just in case we ever change the default in the base class, I thought it'd be good that the default is hard coded and controlled here.

Reply:
That's fine, but if the logic is going to be set explicitly in each subclass does having it in the base serve any purpose?

Reply:
Yeap: dml is not of any of these subclasses and this sets the “auto” for that class for instance